sections/queueing_model/subsections/performance_measures/main.tex

```diff
@@ -0,0 +1,43 @@
+\subsection{Performance Measures}
+
+
+The transition matrix \( Q \) defined in \ref{eq:markov_transition_rate} can be 
```

Comment:
```suggestion
The transition matrix \( Q \) defined in (\ref{eq:markov_transition_rate}) can be 
```

Equations always need to be referred to in brackets. Take a look through as I might have missed some of these.